api/server/server.go

```diff
@@ -554,7 +554,7 @@ func (s *Server) getContainersStats(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	return s.daemon.ContainerStats(vars["name"], boolValue(r, "stream"), ioutils.NewWriteFlusher(w))
+	return s.daemon.ContainerStats(vars["name"], boolValueOrDefault(r, "stream", true), ioutils.NewWriteFlusher(w))
```

Comment:
On my phone ... But was this in 1.6?

We can (and should) just version the API here, < 1.19 == stream always true, else stream is defaulted to false. Will have to update the CLI as well.